app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -0,0 +1,85 @@
+package com.android.streetworkapp.ui.profile
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.R
+import com.android.streetworkapp.ui.navigation.BottomNavigationMenu
+import com.android.streetworkapp.ui.navigation.LIST_TOP_LEVEL_DESTINATION
+import com.android.streetworkapp.ui.navigation.NavigationActions
+
+@Composable
+fun ProfileScreen(navigationActions: NavigationActions) {
+  Scaffold(
+      modifier = Modifier.testTag("ProfileScreen"),
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelect = { route -> navigationActions.navigateTo(route) },
+            tabList = LIST_TOP_LEVEL_DESTINATION,
+            selectedItem = navigationActions.currentRoute())
+      },
+      content = { padding ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(padding).testTag("ProfileColumn"),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.spacedBy(10.dp, Alignment.CenterVertically)) {
+              // profile placeholder
+              Row() {
```

Comment:
Suggestion: I fail to see the utility of using rows for everyone of your nested composables. This does not add anything to the UI and increases code complexity for no reason, I would suggest removing them.
(this concerns line 45, 52, 59, 66 and 74)

Reply:
Indeed, I removed the row